### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by fajaryanw** [axilleasiv/vscode-javascript-repl-docs#21]
I don't know this is a bug or not.
but I just write this code then run JavaScript REPL.
``` Javascript
const profile = {
  firstName: "John",
  lastName: "Doe",
  age: 18
};

const { firstName, lastName, age } = profile;

console.log(firstName, lastName, age);

```
I got a log message `RangeError: Maximum call stack size exceeded`

![image](https://user-images.githubusercontent.com/6999559/95445821-a8143c00-0989-11eb-8c6f-78b356a52e57.png)

**Comment by axilleasiv** [comment]
Hello, @fajaryanw thank you so much for your feedback. It seems that there is a bug!

Although, it works when you are doing `console.log(profile.firstName, profile.lastName, profile.age);` or it works someone logs an entire object `console.log(profile);`, it seems that this is not working during an Object Destructuring and at the moment you can only log only the first property that has been destructured, `console.log(firstName)` in your case. For some reason, it crashes when someone tries to log more than one destructured object properties.

**Comment by axilleasiv** [comment]
Fixed in 0.6.5 version
